test/powershell/Modules/Microsoft.PowerShell.Security/CertificateProvider.Tests.ps1

```diff
@@ -133,6 +143,9 @@ Describe "Certificate Provider tests" -Tags "Feature" {
         }
         it "Should be able to get EnhancedKeyUsageList of certifate by path: <path>" -TestCases $currentUserMyLocations {
             param([string] $path)
+
+            if ($SetupFailure) { Set-ItResult -Inconclusive -Because "Test certificates are not installed"}
```

Comment:
```suggestion
            if ($SetupFailure) {
                Set-ItResult -Inconclusive -Because "Test certificates are not installed"
            }
```